src/petals/cli/run_server.py

```diff
@@ -131,6 +131,7 @@ def main():
     parser.add_argument('--load_in_8bit', type=str, default=None,
                         help="Convert the loaded model into mixed-8bit quantized model. "
                              "Default: True if GPU is available. Use `--load_in_8bit False` to disable this")
+    parser.add_argument("--tensor_parallel_devices", nargs='+', default=[])  # disabled by default
```

Comment:
```suggestion
    parser.add_argument("--tensor_parallel_devices", nargs='+', default=None)
```

Please add a help string here. If you want tensor-parallelism to be enabled by default (I think I'd prefer it), change the default to `None` and remove the comment.